doorstop/cli/main.py

```diff
@@ -201,6 +205,7 @@ def main(args=None):  # pylint: disable=R0915
             if hasattr(settings, i):
                 setattr(settings, i, getattr(file_settings, i))
     else:
+        log.debug(f"Loading default settings")
```

Comment:
Linter doesn't like f-strings without any variables in them. I'm fine with it but drop the 'f' if you want it to pass tests

Reply:
Ah, I have forgotten it there, removed, thanks